vehicle/tests/golden/compile/mnist-robustness/TypeCheck.err.golden

```diff
@@ -0,0 +1 @@
+syntax error at line 1, column 1 before `.'
```

Comment:
Err I'm a bit worried about this. Why is the `mnist-robustness` test now producing an error message when we're type checking? Is this genuine? (i.e. if you delete this file and rerun the test does it generate the same error).

I think there's the same issue in `windController` test.

Reply:
I've done that and the problem stays. It's actually a problem for mnist-robustness, windController, acasXu tests (it seems I haven't pushed the .err files for this one). The tests look the same as in the dev branch, no changes there. For me the .vcl files for these contain a path to the actual spec in /examples (let me know if that shouldn't be the case, it seems some of my problems came from my old pull not deleting certain folders that it should have) - my theory for now is that it can't read them, it errors out on the dot. I've got no idea yet about why they fail for me if they work in general...

Reply:
Ah it's a symlink problem. Summoning @wenkokke! Wen, you said that there shouldn't be any problems with symlinks on Windows, but it looks like there is?

For the meantime @ndslusarz, can you avoid committing these error files and rely on the CI on Github for testing them?